README.md

```diff
@@ -23,8 +23,13 @@ npm install -g yo
 # install the react-server generator
 npm install -g generator-react-server
 
-# make a new react-server project
+# make a new react-server project in the CURRENT directory
```

Comment:
I guess so, but I don't really know.

I also ran into the problem that it defaulted to calling the project by my current directory name, replacing `-` with a space, but when it tried to npm install, that failed. I should file that.